platform/landing-page/src/main/java/org/codice/ddf/landing/LandingPage.java

```diff
@@ -269,8 +245,10 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
     }
 
     // package-private for unit testing
-    String compileTemplateWithProperties() {
-        logoToUse = StringUtils.isNotEmpty(logo) ? logo : productImage;
+    String compileTemplateWithProperties() throws IOException {
+        title = getTitle();
+        version = branding.getAttributeFromBranding(BrandingPlugin::getProductName);
+        logoToUse = StringUtils.isNotEmpty(logo) ? logo : getProductImage();
         // FieldValueResolver so this class' fields can be accessed in the template.
         // MapValueResolver so we can access {{@index}} in the #each helper in the template.
         final Context context = Context.newBuilder(this)
```

Comment:
I think we can simplify this now that we're getting some of the template values from the branding registry. For example, we don't really need the instance variables `title`, `version`, or `logoToUse` anymore since we're calculating them on the fly. We could add a `JavaBeanValueResolver` to the context so we can access values in the template via `get*()` methods in this class.

Then, instead of having to set `title`, `version`, and `logoToUse` at the beginning of the method, `getTitle()` and `getVersion()` would be called to get the values of `title` and `version` in the handlebars template. We'd have to add a getter for `logoToUse`. And we could even add getters for the rest of the fields and we wouldn't need the `FieldValueResolver` anymore.